lambda/samlpost/index.html

```diff
@@ -389,7 +389,8 @@
           secretKey +
           '"\nexport AWS_SESSION_TOKEN="' +
           sessionToken +
-          '"\nexport AWS_DEFAULT_REGION=ca-central-1\n';
+          '"\nexport AWS_DEFAULT_REGION=ca-central-1' +
+          '"\nexport AWS_REGION=ca-central-1\n';
```

Comment:
Remove the leading quotation 

```suggestion
          '\nexport AWS_REGION=ca-central-1\n';
```